mesheryctl/pkg/utils/error.go

```diff
@@ -306,6 +304,52 @@ func ErrInvalidNameOrID(err error) error {
 		[]string{"Provided Name or ID would be invalid"}, []string{"Please Provide a valid name or ID by using list command with the appropriate subcommand"})
 }
 
+func ErrAttachAuthToken(err error) error {
+	return errors.New(ErrAttachAuthTokenCode, errors.Alert, []string{"Authentication token not found"},
+		[]string{"Authentication token not found", err.Error()}, []string{"User not logged in to generate token"}, []string{"Login with `mesheryctl system login` or supply a valid user token using the --token (or -t) flag."})
+}
+
+func ErrFailReqStatus(statusCode int) error {
+	return errors.New(ErrFailReqStatusCode, errors.Alert, []string{"Failed response Server Error"},
+		[]string{"Response Status Code " + strconv.Itoa(statusCode) + ", possible Server Error"}, []string{"Bad api call"}, []string{"Check your network connection and check the status of meshery server via 'mesheryctl system status'"})
+}
+
+func ErrProcessingConfig(err error) error {
+	return errors.New(ErrProcessingConfigCode, errors.Alert, []string{"Error processing config"},
+		[]string{"Error processing config", err.Error()}, []string{"Couldn't load mesh config"}, []string{"Ensure mesheryctl has the right configurations"})
```

Comment:
1. Be consistent in reference to config and "mesh config" - use "meshconfig".
2. "mesh config" is one word: "meshconfig"
3. "configurations" should not be plural.
4. Don't use "right" as the descriptor, but use "valid" instead.